wpilibj/src/test/java/edu/wpi/first/wpilibj/controller/ProfiledPIDControllerTest.java

```diff
@@ -0,0 +1,21 @@
+package edu.wpi.first.wpilibj.controller;
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
+
+import edu.wpi.first.wpilibj.trajectory.TrapezoidProfile;
+
+public class ProfiledPIDControllerTest {
+
+  @Test
+  @SuppressWarnings("PMD")
```

Comment:
Why is PMD mad here? Suppress individual warnings.

Reply:
It's mad about `JUnit tests should include assert() or fail()`

Reply:
But you have an assert?

Reply:
I don't use the Java assert keyword if that's what you're asking.